src/patternfly/components/Switch/styles.scss

```diff
@@ -0,0 +1,119 @@
+@import "../../patternfly-utilities";
+
+.pf-c-switch {
+  --pf-c-switch--Height: calc(#{var(--pf-global--FontSize--md)} * #{var(--pf-global--LineHeight--md)});
+  --pf-c-switch--Width: calc(#{var(--pf-c-switch--Height)} * #{var(--pf-global--GoldenRatio)});
+  --pf-c-switch--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-switch__toggle--Radius: var(--pf-c-switch--Height);
+  --pf-c-switch__toggle--Padding: calc(#{var(--pf-c-switch--Height)} / 2 * (1 - 1 / #{var(--pf-global--GoldenRatio)}));
+  --pf-c-switch__toggle--BackgroundColor: var(--pf-global--primary-color--100);
+  --pf-c-switch__toggle--m-off--BackgroundColor: var(--pf-global--BackgroundColor--dark-200);
+  --pf-c-switch__toggle--m-disabled--BackgroundColor: var(--pf-global--BackgroundColor--dark-transparent-200);
+  --pf-c-switch__thumb--Width: calc(#{var(--pf-c-switch--Height)} / #{var(--pf-global--GoldenRatio)});
```

Comment:
should these instances of the `thumb` element be `toggle` instead? if not, what does `thumb` represent?